experiments/experiment_data.py

```diff
@@ -93,6 +97,7 @@ def _update_best_trackers(self, run_config, run_config_measurement):
     def _get_run_config_measurement_from_keys(self, ma_key, pa_key):
         if ma_key not in self._data:
             print(f"WARNING: Model config {ma_key} not in results")
+            self._missing_measurement_count += 1
             return
         if pa_key not in self._data[ma_key]:
             return None
```

Comment:
any reason we don't increment missing measurement count here?

Reply:
I agree. We should increment here as well.